tools/app/app/components/CardContextMenu.tsx

```diff
@@ -12,24 +12,18 @@
 
 import * as React from 'react';
 import MoreIcon from '@mui/icons-material/MoreHoriz';
-
-import { CardMetadata } from '../lib/definitions';
 import {
   MenuButton,
   Menu,
   MenuItem,
   Divider,
   Typography,
   Dropdown,
+  Tooltip,
```

Comment:
In general, it would be helpful to keep the imports in alphabetical order, if possible. 
Makes finding the needed item when reading the code a bit easier.

Reply:
Wonder if we could get Prettier to do that automatically

Reply:
Dev depencies are "ok". Doesn't really matter, if it makes our life easier.